.github/workflows/continuous-integration.yml

```diff
@@ -218,7 +218,9 @@ jobs:
       matrix:
         php-version:
           - "7.4"
-          - "8.4"
+          - "8.3"
+          # The DB2 workflow currently segfaults with PHP 8.4
```

Comment:
It would be great to have a bug report to link to here

Reply:
Linking _a_ bug report for reference: https://github.com/shivammathur/setup-php/issues/992.